<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseInsertPage</class>
 <widget class="QDialog" name="DatabaseInsertPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>291</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="id_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="pilenumber_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="responsible_person_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="date_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="charge_count_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="rod_count_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="detonator_count_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="well_depth_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="charge_count_detected_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="rod_count_detected_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="manual_inspection_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="status_2"/>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="id">
        <property name="text">
         <string>id</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pilenumber">
        <property name="text">
         <string>pilenumber</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="responsible_person">
        <property name="text">
         <string>responsible_person</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="date">
        <property name="text">
         <string>date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="charge_count">
        <property name="text">
         <string>charge_count</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rod_count">
        <property name="text">
         <string>rod_count</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="detonator_count">
        <property name="text">
         <string>detonator_count</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="well_depth">
        <property name="text">
         <string>well_depth</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="charge_count_detected">
        <property name="text">
         <string>charge_count_detected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rod_count_detected">
        <property name="text">
         <string>rod_count_detected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="manual_inspection">
        <property name="text">
         <string>manual_inspection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="status">
        <property name="text">
         <string>status</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>222</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>58</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="verifyButton">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
